.github/actions/setup-project/action.yml

```diff
@@ -7,9 +7,6 @@ inputs:
   node-version:
     required: false
     default: 18
-  turbo-version:
```

Comment:
Does removing this void any cache it may store?

Reply:
I don't think this was necessary, as `pnpm` installs it for us? And I believe the packages are cached already in our node setup action. That said, I'm no Turbo expert